tests/Doctrine/Tests/ORM/Functional/SchemaTool/MySqlSchemaToolTest.php

```diff
@@ -89,6 +105,89 @@ public function testGetCreateSchemaSql4()
         $this->assertEquals(0, count($sql));
     }
 
+    public function testUpdateSchemaSql()
+    {
+        $classes = [
+            $this->_em->getClassMetadata(MyEntityToRemove::class),
+        ];
+        $tool    = new SchemaTool($this->_em);
+        $sqls    = $tool->getUpdateSchemaSql($classes);
+        $sqls    = $this->filterSqls($sqls, ['entity_to_remove']);
+        $this->assertCount(1, $sqls);
+        $this->assertContains('CREATE TABLE entity_to_remove (id INT AUTO_INCREMENT NOT NULL, PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB', $sqls);
```

Comment:
`self::assertContains()` since the method is static (other places should be fixed as well)

Reply:
I'm afraid this will also fail on DBAL 2.10+ due to quoted collation